1-js/02-first-steps/03-strict-mode/article.md

```diff
@@ -1,89 +1,89 @@
+Nu există o comandă de tipul `"no use strict"` care să readucă motorul de căutare ("browserul") la vechiul comportament.
 
-Once we enter strict mode, there's no going back.
+Odată ce am activat modul strict, nu mai există cale de întoarcere.
 ```
 
-## Browser console
+## Consola browserului
 
-When you use a [developer console](info:devtools) to run code, please note that it doesn't `use strict` by default.
+Atunci când folosiți [developer console](info:devtools) pentru a rula codul, vă rugăm să rețineți că aceasta nu folosește `use strict` în mod implicit.
 
-Sometimes, when `use strict` makes a difference, you'll get incorrect results.
+Uneori, când `use strict` face o diferență, veți obține rezultate incorecte.
 
-So, how to actually `use strict` in the console?
+Așadar, cum se utilizează de fapt `use strict` în consolă?
 
-First, you can try to press `key:Shift+Enter` to input multiple lines, and put `use strict` on top, like this:
+În primul rând, puteți încerca să apăsați `key:Shift+Enter` pentru a introduce mai multe linii, și să puneți `use strict` deasupra, astfel:
 
 ```js
-'use strict'; <Shift+Enter for a newline>
-//  ...your code
-<Enter to run>
+'use strict'; <Shift+Enter pentru o linie nouă>
+//  ...codul tău
+<Enter pentru a executa>
 ```
 
-It works in most browsers, namely Firefox and Chrome.
+Funcționează în majoritatea browserelor, mai ales în Firefox și Chrome.
 
-If it doesn't, e.g. in an old browser, there's an ugly, but reliable way to ensure `use strict`. Put it inside this kind of wrapper:
+Dacă nu funcționează, de exemplu, într-un browser vechi, există o modalitate inestetică, dar fiabilă de a asigura `use strict`. Puneți-l în acest tip de înveliș:
 
 ```js
 (function() {
   'use strict';
 
-  // ...your code here...
+  // ...codul tău...
 })()
 ```
 
-## Should we "use strict"?
+## Ar trebui să folosim comanda "use strict"?
 
-The question may sound obvious, but it's not so.
+Întrebarea poate părea banală, dar nu este așa.
 
-One could recommend to start scripts with `"use strict"`... But you know what's cool?
+Cineva ar putea recomanda să începeți scripturile cu `"use strict"`... Dar știi ce este grozav?
 
-Modern JavaScript supports "classes" and "modules" - advanced language structures (we'll surely get to them), that enable `use strict` automatically. So we don't need to add the `"use strict"` directive, if we use them.
+JavaScriptul modern suportă "classes" ("clase") și "modules" ("module") - structuri avansate de limbaj (vom ajunge cu siguranță la ele), care permit `use strict` în mod automat. Astfel, nu este nevoie să adăugăm comanda `"use strict"`, dacă le folosim.
 
-**So, for now `"use strict";` is a welcome guest at the top of your scripts. Later, when your code is all in classes and modules, you may omit it.**
+**Deci, deocamdată `"use strict";` este binevenit în partea de sus a scripturilor dumneavoastră. Mai târziu, când tot codul dumneavoastră va cuprinde "classes" și "modules", îl veți putea omite.**
```

Comment:
```suggestion
**Deci, deocamdată `"use strict";` este un oaspete binevenit în partea de sus a scripturilor dumneavoastră. Mai târziu, când tot codul tău va area classes și modules peste tot, îl veți putea omite.**
```

Nu pare că articolul este așa formal să folosim dumneavoastră. Restul schimbărilor pot fi subiective.

Reply:
> Da, pentru că în engleză nu ai pronume de politețe :)

Dacă nu s-au folosit în textul original, nu folosim nici noi decăt dacă nu face sens altfel.

> Am omis intenționat ”oaspete”, suna dubios.

Și mie îmi sună puțin dubios dar nu știam dacă este altceva echivalent în limba română care să aibă sens. Decât dacă eliminăm expresia cu totul așa cum ai propus tu... Deși încerc șă mă abat cât mai puțin de la textul original să nu-i schimb sensul.

Reply:
> ”Mai târziu, când tot codul vostru va cuprinde "classes" și "modules"

Sună ok, poți face modificarea și îl trimitem așa. Singurul lucru este că textul original nu are ghilimele pentru classes și modules.

Reply:
Am facut un pull request pe engleză.
Am propus ”So, for now `"use strict";` is a **good literal expression** at the top of your scripts. Later, when your code is all in classes and modules, you may omit it.”

Reply:
S-ar putea să dureze ceva timp până sugestia o să fie adresată. Pentru îmbunătățiri, primim toate actualizările de la versiunea din engleză *automat*. Asta face integrarea lor mult mai usoară. Deci putem traduce cum putem, iar dacă sunt schimbări le încorporăm mai târziu.